plugin/core/panels.py

```diff
@@ -109,6 +109,8 @@ def run(self, edit: sublime.Edit, characters: Optional[str] = "") -> None:
 class LspUpdateServerPanelCommand(sublime_plugin.TextCommand):
     def run(self, edit: sublime.Edit, prefix: str, message: str) -> None:
         with mutable(self.view):
+            message = message.replace("\r\n", "\n")  # normalize Windows eol
+            message = message.replace("\r", "\n")  # normalize MacOS eol
```

Comment:
These `\r` from macOS seem ancient. I don't think anybody is using macOS 9 anymore?

https://superuser.com/questions/439440/did-mac-os-lion-switch-to-using-line-feeds-lf-n-for-line-breaks-instead-of